### ISSUE TITLE
google-chrome

### ISSUE THREAD

**Issue by frhgit** [juliensalinas/torrengo#15]
Hello,
First thank you for your great work on torrengo.

I'm using it on a raspberry/arch linux
I did 
go get -v -u github.com/juliensalinas/torrengo
go build -v github.com/juliensalinas/torrengo
I tried it and I got the following error
error="error while fetching url: could not download page: exec: \"google-chrome\": executable file not found in $PATH

I tried to install google-chrome but google-chrome is not available for the 'armv7h' architecture.

Did I miss something?
could you add a switch to come back to the older behavior?

Regards

**Comment by juliensalinas** [comment]
@frhgit thanks for the feedback. I just made this switch a couple of days ago for the following reason: today neither The Pirate Bay, nor Ygg Torrent, nor 1337 are working without a proper way of handling Javascript as they are all using the Cloudflare anti-bot protection.
I looked into several options but it seemed that the best one was to use a real Google Chrome browser under the hood.

Unfortunately without Google Chrome this program is pretty useless as almost all the torrent websites will block the program...

If you think of another interesting option feel free to suggest!

**Comment by frhgit** [comment]
Dear Julien
Thank you for taking your time to answer me.
I totally understand what you made the switch. I think it is a very good improvement.

for your information 
- On my raspberry/arch linux, ygg and  1337 were working with the previous version (maybe due to python-cloudlfare package that come with arch linux?)
- I tried to install your software on a debian but result was a panic runtine error
- I setup a virtualbox with arch linux and it worked perfectly.

I'm sorry I have not your knowledge in software development, so I don't see another option to replace google-chrome except to check the python-cloudflare version.

**Comment by juliensalinas** [comment]
@frhgit I dropped the Python Cfscrape dependency in January this year and replaced it with a Go version of Cfscrape. It worked for 6 months but now I had to replace it again (with ChromeDP).
If you want to go back to the Python Cfscrape version, no problem, the last commit before the replacement is https://github.com/juliensalinas/torrengo/commit/609e11d10000281d805aa6fec12e93284a70fd5d
You would need to compile a version of Torrengo based on this old commit.

Thanks for letting me know about Debian. I never tried to run Torrengo on Debian but I find it very strange that it's not working on a Debian distrib... I'll have a look when I have a moment.

Cheers
